source/libphone/libphone.cpp

```diff
@@ -0,0 +1,111 @@
+#include "libphone.hpp"
+#include "utils/utils.hpp"
+
+namespace LibPhone {
+Phone::Phone(zenohc::Session& session)
+	: m_app(uWSAppWrapper({.passphrase = "1234"})),
+	  m_supported_instruments({"bnb_usdt_spot", "busd_usd_spot"}),
```

Comment:
This is the only set of supported instruments

Reply:
that, as @arjunaskykok suggested, will be fetched from an env variable. Another fix for the final stable release